WTWSMS/common/cultures/00_cultures.txt

```diff
@@ -4644,7 +4645,13 @@ byzantine = {
 		
 		founder_named_dynasties = yes
 
-		modifier = default_culture_modifier
+		allow_looting = yes
+
+		character_modifier = {
+			martial = 4
```

Comment:
Why would it apply to other cultures too? You haven't understood culture modding: you can't add these bonuses to a group, it is per culture only.